Savanna/Model/Antelope.cs

```diff
@@ -0,0 +1,127 @@
+﻿namespace Savanna
+{
+    public class Antelope : Animal
+    {
+        /// <summary>
+        /// Antilope entity.
+        /// </summary>
+        public Antelope()
+        {
+            Id = GenerateId();
+            Vision = GameParameters.vision;
+            IsAlive = true;
+            Letter = GameMessages.antilopeLetter;
+        }
+
+        /// <summary>
+        /// Moves current animal on the game field.
+        /// </summary>
+        /// <param name="game">Current game field.</param>
+        /// <param name="animal">Current animal.</param>
+        /// <param name="animals">List of animals.</param>
+        public override void MoveAnimal(Game game, Animal animal, List<Animal> animals)
+        {
+            Antelope antelope = (Antelope)animal;
+            List<Lion> lionsAround = AnimalsInRange<Lion>(antelope, animals);
+            Animal? nearestLion = FindNearestLion(lionsAround, antelope);
+            List<AnimalNewCoordinates> newAnimalsCoordinates = FreeCellsToMove(animals, antelope);
+
+            if (nearestLion == null)
+            {
+                MakeNextRandomMove(antelope, newAnimalsCoordinates);
+            }
+            else
+            {
+                RunAway(nearestLion, antelope, newAnimalsCoordinates);
+            }
+        }
+
+        /// <summary>
+        /// Gets closest lion with all parameters to current antelope.
+        /// </summary>
+        /// <param name="lionsAround">List of lions in current animal vision range.</param>
+        /// <param name="antilope">Current animal.</param>
+        /// <returns></returns>
+        private Animal? FindNearestLion(List<Lion> lionsAround, Animal antelope)
```

Comment:
FindNearestLion and CalculateClosestLion are similar by naming and return/input parameters. Combine these methods into one.

Reply:
Methods have been merged.